packages/core/crates/core-mixer/Cargo.toml

```diff
@@ -0,0 +1,36 @@
+[package]
+name = "core-mixer"
+version = "0.1.0"
+authors = ["Tibor Csóka"]
+edition = "2021"
+
+[lib]
+crate-type = ["cdylib", "rlib"]
+
+[features]
+default = ["console_error_panic_hook", "wasm"]
+wasm = []
+
+[dependencies]
+async-std = { version = "1.12.0", features = ["attributes"] }
+futures = "0.3.26"
+futures-lite = "1.12.0"
+rand = "0.8.5"
+pin-project = "1.0.12"
+# wasm specific
+utils-misc = { path = "../../../utils/crates/utils-misc" }
+getrandom = { version = "0.2.8", features = ["js"] }
```

Comment:
```suggestion
getrandom = { version = "0.2.8", features = ["js"], optional = true }
```
only fetches / compile package if `wasm` feature got activated

Reply:
https://github.com/hoprnet/hoprnet/actions/runs/4175413790/jobs/7230293598

Reply:
As @robertkiel mentioned, all wasm specific dependencies should be `optional = true` and enabled only when `wasm` feature is on.